src/seaport.ts

```diff
@@ -1029,12 +1029,13 @@ export class OpenSeaPort {
       from = proxyAddress
     }
 
-    try {
+    const data = encodeTransferCall(abi, fromAddress, toAddress)
```

Comment:
Why did you move this here?

Reply:
I realized that if `encodeTransferCall` throws, then it shouldn't suppress the error. if `estimateGas` throws then the asset is not transferrable, and we should return false. but if `encodeTransferCall` throws then it's probably a developer error